docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -186,7 +186,8 @@ Name                                                         Description
                                                              For example, ``abc:xyz;example.foo:bar`` sets the system property
                                                              ``abc`` to the value ``xyz`` and the ``foo`` property for
                                                              catalog ``example`` to the value ``bar``.
-``externalAuthentication``                                   Use a local web browser to authenticate with an identity provider (IdP)
+``externalAuthentication``                                   Set to true if you want to use external authentication via OAuth 2.0.
```

Comment:
Change this to make OAuth2 in this sentence a link and then get rid of the last sentence in this description (See...)